flink-jepsen/docker/Dockerfile-db

```diff
@@ -35,5 +35,12 @@ RUN mkdir -p /root/.ssh/ && \
     chmod 600 /root/.ssh/authorized_keys && \
     cat /root/id_rsa.pub >> /root/.ssh/authorized_keys
 
+COPY sshd-run /etc/sv/service/sshd/run
+RUN chmod +x /etc/sv/service/sshd/run && \
+    ln -sf /etc/sv/service/sshd /etc/service
+
 EXPOSE 22
-CMD exec /usr/sbin/sshd -D
+
+# Start runit process supervisor which will bring up sshd.
+# In our tests we can use runit to supervise more processes, e.g., Mesos.
+CMD runsvdir -P /etc/service /dev/null > /dev/null
```

Comment:
Is the extra `> /dev/null` actually required? I would have expected that the log argument to `/dev/null` alone would have sufficed, since it also redirects standard error according to the docs.

Reply:
Yes is needed. It only redirects std err:

>>> If the log argument is given to runsvdir, all output to standard error is redirected to this log